eng/build.ps1

```diff
@@ -194,7 +194,7 @@ if ($Help) {
 
 if ($DumpProcesses -or $CI) {
     # Dump running processes
-    Start-Job -Name DumpProcesses -FilePath $PSScriptRoot\eng\scripts\dump_process.ps1 -ArgumentList $PSScriptRoot
+    Start-Job -Name DumpProcesses -FilePath $PSScriptRoot\scripts\dump_process.ps1 -ArgumentList $PSScriptRoot
```

Comment:
Need to fix https://github.com/dotnet/aspnetcore/blob/d82046a8648a65fd1f99fda81ed5376a175902eb/eng/build.ps1#L308 too